src/views/blocks/tickets/submit-button-modal.php

```diff
@@ -31,12 +32,38 @@
 $content     = apply_filters( 'tribe_events_tickets_attendee_registration_modal_content', '<p>Ticket Modal</p>', $this );
 
 $args = [
-	'button_classes' => [ 'tribe-common-c-btn--small tribe-block__tickets__submit' ],
-	'button_name'    => $provider_id . '_get_tickets',
-	'button_text'    => $button_text,
-	'button_type'    => 'submit',
-	'show_event'  => 'tribe_dialog_show_ar_modal',
-	'title'          => $title,
+	'append_target'           => 'body',
+	'content_wrapper_classes' => 'tribe-common tribe-dialog__wrapper tribe-modal__wrapper--ar',
+	'button_classes'          => [ 'tribe-common-c-btn', 'tribe-common-c-btn--small', 'tribe-tickets__submit' ],
+	'button_name'             => $provider_id . '_get_tickets',
+	'button_text'             => $button_text,
+	'button_type'             => 'submit',
+	'show_event'              => 'tribe_dialog_show_ar_modal',
+	'title'                   => $title,
+	'title_classes'           => [
+		'tribe-dialog__title',
+		'tribe-modal__title',
+		'tribe-common-h5',
+		'tribe-common-h--alt',
+		'tribe-modal--ar__title',
+	],
 ];
 
 tribe( 'dialog.view' )->render_modal( $content, $args );
+$event_id = get_the_ID();
+/** @var Tribe__Tickets__Editor__Template $template */
+$template = tribe( 'tickets.editor.template' );
+$provider_id = Tribe__Tickets__Tickets::get_event_ticket_provider( $post_id );
+$provider    = call_user_func( [ $provider_id, 'get_instance' ] );
+$obj_tickets = $provider->get_tickets( $event_id );
+foreach( $obj_tickets as $ticket ) {
+	$ticket_data = [
+		'id'       => $ticket->ID,
+		'qty'      => 1,
+		'provider' => $provider,
+	];
+
+	$tickets_content[] = $ticket_data;
+}
+
+$template->template( 'registration-js/attendees/content', array( 'event_id' => $event_id, 'tickets' => $tickets_content ) );
```

Comment:
Can we move this new logic out of the view and pass the `tickets`?

Reply:
hm, maybe...I believe so yes.